packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -271,7 +272,7 @@ const ProofOfAddressForm = observer(
         }
         const setOffset = (status: { msg: string }) => {
             const mobile_scroll_offset = status?.msg ? '200px' : '154px';
-            return is_mobile && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
+            return (isMobile || isTablet) && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
```

Comment:
just curious, why not !isDesktop 😄 ?

Reply:
Ohh yes. let me update this as well. 😬 
Thanks @utkarsha-deriv for highlighting